src/js/components/Paragraph/doc.js

```diff
@@ -29,3 +29,63 @@ export const doc = Paragraph => {
 
   return DocumentedParagraph;
 };
+
+export const themeDoc = {
+  'global.colors.text': {
+    description: 'The text color used inside the Paragraph.',
+    type: 'string | { dark: string, light: string }',
+    defaultValue: "{ dark: '#f8f8f8', light: '#444444' }",
+  },
+  'global.text': {
+    description: 'The text size of the Paragraph text.',
+    type: 'string',
+    defaultValue: 'medium',
+  },
+  'global.edgeSize.small': {
```

Comment:
I don't think we need this as well as `global.edgeSize` below. I recommend dropping this one and keeping `global.edgeSize`.